aicsimageio/readers/tiff_reader.py

```diff
@@ -51,25 +52,20 @@ def __init__(self, image: types.PathLike):
 
         # Enforce valid image
         if not self._is_supported_image(self.fs, self.path):
-            raise exceptions.UnsupportedFileFormatError(self.extension)
+            raise exceptions.UnsupportedFileFormatError(
+                self.__class__.__name__, self.extension
+            )
 
     @property
-    def scenes(self) -> Tuple[int]:
+    def scenes(self) -> Tuple[str]:
         if self._scenes is None:
             with self.fs.open(self.path) as open_resource:
                 with TiffFile(open_resource) as tiff:
                     # This is non-metadata tiff, just use available series indicies
-                    self._scenes = tuple(range(len(tiff.series)))
+                    self._scenes = tuple(f"Image:{i}" for i in range(len(tiff.series)))
```

Comment:
because "Image:index" is ome-specific and I've seen it hard coded in 2 places now, I would consider writing a little utility function ome_scene_name(index) that returns that string.